### DISCUSSION TITLE
Unhold icon

### DISCUSSION THREAD

**Discussion by robyn-bautista** [twilio-labs/paste#3258] [General] [answered]
Hi team! Following up on [this thread](https://twilio.slack.com/archives/C040QETJ0Q1/p1684425770044999) to make sure our hold design is paste compliant

Agreed that the toggle button should be used as is - my question is around giving more affordance via an icon change. We've used a hold/unhold icon, but I'm not seeing it in paste. Another option is using the play button, although it doesn't seem as common a pattern. 

https://www.figma.com/file/3tabCGxnI3s1SrdCU0sZbV/Aion---Omnichannel-Canvas?type=design&node-id=137%3A154674&t=BfWwQ0YrpdxtFLsZ-1

Thoughts?

**Comment by libertycarlin** [comment]
Hi @robyn-bautista after taking a look at your designs I feel like you could potentially use a hold and play button as long as your using the toggle button states. Curious thoughts around the need for an icon change. When using the toggle states It seems clear to me when one is selected vs. not selected. Here's me playing in figma :) https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?type=design&node-id=2627%3A629476&t=KE9VOyOpMG2MHSkh-1

Do you guys have any research on why there's a need for an icon change at all? 

Talk soon!

**Reply by robyn-bautista** [reply]
Hey Liberty, thanks for following up!

[removed] For what it’s worth, we have plans to add in helper text of “call on hold” which will help too

I also noticed there is an off state for microphone (such as for mute) and for a few others. We’re implementing mute as well so what are your thoughts about having consistent behavior of an icon change across the board for media controls?

Example:

[removed]

**Reply by loreina** [reply, marked as answer]
Hi @robyn-bautista,

Thanks for the video, it's really helpful to see the interaction in context.

We'd recommend first switching to use [Toggle Button](https://paste.twilio.design/components/button#toggle-button) and testing that with customers for feedback. We initially added this component for Flex/Aion and to solve for this use case.

If there are still concerns from customers, let us know and we'll reconsider the design for Toggle Button and adding additional call icons as this interaction is exactly what we expect the component to solve for and why we added it to Paste. 

Can you also share examples you've found of call interactions in other products? Thanks!

**Reply by robyn-bautista** [reply]
Thanks for the help! Our approach will be to first update the helper text to say "Call on hold" and have a loading state for any delays.
As a follow up, we will consider the pause and play icons like shown [here](https://paste.twilio.design/components/button#toggle-button) for pause/resume call and get user feedback
